docs/setup/native.md

```diff
@@ -54,6 +54,26 @@ Vets API requires:
 
 1. Run `bin/setup` to setup the database and start the server.
 
+### pg_stat_statements
+If you have trouble enabling query stats from the PgHero dashboard, try enabling it manually
+
+Add the lines below to your main postgresql.conf file
+
+On Mac it should be located somewhere similiar to the following:
+
+`/Users/[your username]/Library/Application Support/Postgres/var-12/postgresql.conf`
+```
+shared_preload_libraries = 'pg_stat_statements'
+pg_stat_statements.track = all
+pg_stat_statements.max = 10000
+track_activity_query_size = 2048
+```
+Then from the command line:
+`psql -U postgres`
+
+Next from the psql console:
+`CREATE extension pg_stat_statements;`
```

Comment:
We should have a migration to do this instead of having non-standard procedures to follow.

Reply:
I can remove this piece I think. We do have a migration already https://github.com/department-of-veterans-affairs/vets-api/blob/master/db/migrate/20210507122840_add_stats_extension.rb

Reply:
Fixed ✔️